src/installer/tests/HostActivation.Tests/NativeHosting/HostContext.cs

```diff
@@ -237,6 +238,31 @@ public void GetDelegate_Multiple(string checkProperties)
             propertyValidation.ValidateSecondaryContext(result, SharedTestState.SecondaryConfigPropertyName, SharedTestState.SecondaryConfigPropertyValue);
         }
 
+        [Fact]
+        public void GetDelegate_Active()
+        {
+            string newPropertyName = "HOST_TEST_PROPERTY";
+            string[] args =
+            {
+                HostContextArg,
+                Scenario.GetRuntimeDelegate,
+                CheckProperties.None,
+                sharedState.HostFxrPath,
+                sharedState.RuntimeConfigPath,
+                SharedTestState.ConfigPropertyName,
+                newPropertyName
+            };
+            CommandResult result = sharedState.CreateNativeHostCommand(args, sharedState.DotNetRoot)
+                .Execute();
+
+            result.Should().Pass()
+                .And.InitializeContextForConfig(sharedState.RuntimeConfigPath)
+                .And.CreateDelegateMock_COM();
```

Comment:
This should validate the `hdt_load_in_memory_assembly` delegate, as that is the one retrieved with `nullptr` context.

Reply:
Done